.gitignore

```diff
@@ -25,6 +25,8 @@ yarn-debug.log*
 yarn-error.log*
 
 # local env files
+.env
+.gitignore
```

Comment:
I see you added `.gitignore` inside `.gitignore`, was that intentional?

Reply:
> I see you added `.gitignore` inside `.gitignore`, was that intentional?

my bad, I just wanted to add env but added gitignore also. my bad